src/templates/rpc/Dockerfile_rpc

```diff
@@ -1,17 +1,26 @@
 # Use an official Python runtime as the base image
-FROM python:3.11-slim
+FROM python:3.12-slim
 
 # Install procps, which includes pgrep
 RUN apt-get update && \
-    apt-get install -y procps openssh-client && \
+    apt-get install -y curl procps openssh-client && \
     rm -rf /var/lib/apt/lists/*
 
+# Install `uv` packge installer (https://github.com/astral-sh/uv)
+ADD --chmod=755 https://astral.sh/uv/install.sh /install.sh
```

Comment:
I would prefer to install via `pip`. Lots of tools seem to be moving to install scripts via dns, but it makes me worried about man-in-the-middle, astral.sh forgetting to renew their domain, etc.

While `pip` may not verify anything by default, I think using `pip` now makes it easier to transition towards [secure installs](https://pip.pypa.io/en/stable/topics/secure-installs/)

Reply:
Fine with me :)

Reply:
Done in b764c211c32f20b9a225804191d99c5a33dc995b